scripts/install-autoshutdown-server-extension/on-jupyter-server-start.sh

```diff
@@ -40,10 +40,9 @@ else:
 EOF
 chmod +x .auto-shutdown/set-time-interval.sh
 
-# "wget" is not part of the base Jupyter Server image, you need to install it first if needed to download the tarball
-sudo yum install -y wget
 # You can download the tarball from GitHub or alternatively, if you're using VPCOnly mode, you can host on S3
-wget -O .auto-shutdown/extension.tar.gz https://github.com/aws-samples/sagemaker-studio-auto-shutdown-extension/raw/main/sagemaker_studio_autoshutdown-0.1.5.tar.gz
+curl -L https://github.com/aws-samples/sagemaker-studio-auto-shutdown-extension/raw/main/sagemaker_studio_autoshutdown-0.1.5.tar.gz --output .auto-shutdown/extension.tar.gz
```

Comment:
I would go even as far as running it in silent mode as we don't need that progress bar. 😉 
```suggestion
curl -s -L https://github.com/aws-samples/sagemaker-studio-auto-shutdown-extension/raw/main/sagemaker_studio_autoshutdown-0.1.5.tar.gz --output .auto-shutdown/extension.tar.gz
```